util/include/crypto/openssl/crypto.hpp

```diff
@@ -40,6 +40,8 @@
 
 namespace concord::crypto::openssl {
 
+void _openssl_free(void* ptr);
```

Comment:
Document why the function was declared when the OPENSSL_FREE macro already exists.

Reply:
added